lib/actions/artisan.js

```diff
@@ -184,19 +186,63 @@ module.exports = {
       return;
     }
 
-    // Log
-    Logs.show('server-started', {
-      type: 'Info',
-      message: null,
-      params: {}
+    // Grab the configurations
+    const config = JSON.parse(Utils.readFileSync('./site.json', 'utf-8'));
+    const portToSet = options.port || config.server.port || env.server.defaultPort;
+
+    // If the given port is different than whats in the config, update site.json with the new port
+    if (portToSet !== config.server.port) {
+      config.server.port = portToSet;
+      fsExtra.outputFileSync(`${env.templates.config}`, JSON.stringify(config, null, 2));
+    }
+
+    // Setup the database connection
```

Comment:
How about moving this logic into `actions/db.js`?
We can add new method as `db.queryExec()` or like that.